.github/workflows/coverage_report.yaml

```diff
@@ -0,0 +1,36 @@
+name: coverage_report
+
+on:
+  schedule:
+    - cron: '0 0 * * 5'
+
+jobs:
+  coverage_report:
+    name: coverage_report
+    runs-on: [self-hosted,Linux]
+    if: |
+        github.event_name == 'schedule' &&
+        github.repository_owner == 'nervosnetwork'
+    steps:
+      - name: install nightly
+        run: |
+          rustup toolchain list | { grep 'nightly' || rustup install nightly; }
+      - uses: actions/checkout@v2
+      - name: unit coverage
+        run: make cov
+      - name: integration cov
+        run: make integration-cov
+      - name: upload unit cov result to codecov.io
+        uses: codecov/codecov-action@v2
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Not required for public repo [※](https://github.com/codecov/codecov-action#:~:text=%23%20not%20required%20for%20public%20repos)